### ISSUE TITLE
Fix To Do label color contrast - black text is hard to read

### ISSUE THREAD

**Issue by laurentenhoor** [laurentenhoor/devclaw#423]
The "To Do" label has poor contrast between the background color and black text, making it difficult to read. Need to either darken the background color or switch to lighter text for better accessibility.
